### ISSUE TITLE
incorrect EMC values for Extra Utilities 2 (With TL;DR)

### ISSUE THREAD

**Issue by AChubbyPinkFluffyFabWhale** [AshenGaming/potentia-reborn#4]
These items from Extra utilities 2 doesn't have the correct EMC value
1.Stone Drum = 13 EMC cost (6 cobblestone = 6 EMC total, 1 bowl = 6 EMC total, 2 stone/cobblestone slabs doesn't have EMC value 6+6 = 12 EMC cost the raw items)
2.Stone Spikes (per 4) = 28 EMC cost ( 3 stone swords = 18 EMCtotal, 3 cobblestone = 3 EMCtotal, 1 compressed cobblestone = 9 EMC total. 18+3+9= 30 EMC cost in raw material)
3.Antenna = 1663 EMC cost (2 end rods = 864 EMC total, 3 iron ingot =768 EMC total, 1 random sapling = 32 EMC total, 864+768+32 = 1664 EMC cost in raw material)
4.Transfer pipe = 1 EMC cost ( 6 stone slabs = Doesn't have a EMC value, 2 stained glass = 2 EMC total, 1 Redstone = 64 EMC total, 64+2 = 68 EMC in raw material)
5.Trash Can = 71 EMC cost, (5 cobblestone = 5 EMC total, 1 chest = 63 EMC total, 3 stone = 3 EMC total,
63+5+1 = 69 total cost in raw material)
6.Transfer Filter (per 4) = 412 EMC cost, (2 stone = 2 EMC total, 1 Transfer pipe = 1 EMC cost, 1 filter = 284 EMC, 2 redstone = 128 EMC cost, 2+1+284+128 = 415 EMC cost in raw material)
7.Transfer node (Fluids per 4), =896 EMC,  2 redstone = 128 EMC cost, 1 transfer pipe = 1 EMC cost, 2 Stone = 2 EMC cost, 1 bucket =768, 128+2+2+768= 900 cost in Raw material)
8.Transfer node (Items per 4) = 192 EMC, 2 redstone = 128 EMC, 2 stone = 2 EMC, 1 chest = 63 EMC, 1 9.transfer pipe = 1 EMC, 128+63+1+1=194 total in raw material)
9.transfer pipe filter (per 4) = 348 EMC, 1 Redstone = 64 EMC, 1 transfer pipe = 1 EMC, 1 Filter = 284, 64+1+284 = 349 EMC in raw material
TLDR:
1. Stone Drum 1 EMC too low
2. Stone spikes (per 4) 2 EMC too low
3. Atenna 1 EMC too low 
4. Transfer Pipe 67 too low
5. Trash Can 2 too low
6. Transfer filter (per 4) 3 too low
7. Transfer node (fluids per 4) 4 too low
8. Transfer node (Items per 4) 4 too low
9. Transfer pipe filter (per 4) 1 too low

**Comment by AshenGaming** [comment]
EMC values for XU2 will be removed in 0.3.
